src/libcore/default.rs

```diff
@@ -84,10 +84,33 @@
 
 use marker::Sized;
 
-/// A trait for giving a type a useful default value.
+/// A trait for giving a type a useful default value. For more information, see
+/// [the module-level documentation][module].
```

Comment:
Ooooh I'd love moving the module-level docs into here! I think this is where you'd likely end up after getting an error message that you needed a trait and searching in the docs for it.